traffic_ops/traffic_ops_golang/parameter/parameters.go

```diff
@@ -175,8 +175,14 @@ func (param *TOParameter) Read(h http.Header, useIMS bool) ([]interface{}, error
 		if err = rows.StructScan(&p); err != nil {
 			return nil, nil, errors.New("scanning " + param.GetType() + ": " + err.Error()), http.StatusInternalServerError, nil
 		}
-		if p.Secure != nil && *p.Secure && param.ReqInfo.User.PrivLevel < auth.PrivLevelAdmin {
-			p.Value = &HiddenField
+		if p.Secure != nil && *p.Secure {
+			if param.ReqInfo.Config.RoleBasedPermissions {
```

Comment:
`RoleBasedPermissions` is only supposed to have an effect API versions 4 and later, and this handler is shared across all versions.